tests/kernel/sched/schedule_api/testcase.yaml

```diff
@@ -4,6 +4,8 @@ common:
 tests:
   kernel.scheduler:
     filter: not CONFIG_SCHED_MULTIQ
+    # FIXME: See issue #39948
+    platform_exclude: qemu_cortex_a53_smp
```

Comment:
Not sure if this is one of those "intermittently failing tests," but it likely is.

Reply:
Yeah, I think that's the case here.